### DISCUSSION TITLE
Deepl

### DISCUSSION THREAD

**Discussion by quentinglorieux** [sveltia/sveltia-cms#86] [Q&A]
Impossible to see Deepl settings to enter the API in v 0.9.4 ?
Where should it be ?
Is it available on local repository mode ?

**Comment by kyoshino** [comment]
See https://github.com/sveltia/sveltia-cms/tree/main#use-deepl-to-translate-entry-fields

It is available both in remote and local mode.

**Comment by quentinglorieux** [comment]
I have an issue with all integrations actually.
No save button or activation button and no Deepl field ?
Is there something to activate in the config file ?
<img width="840" alt="Screenshot 2023-12-20 at 17 45 55" src="https://github.com/sveltia/sveltia-cms/assets/16529637/68afa944-386b-4857-9cc6-c7741014fb57">

**Comment by kyoshino** [comment]
Ah yes! You need to [configure i18n](https://decapcms.org/docs/beta-features/#i18n-support). The DeepL setting will appear in the Languages tab when there are 2 or more `locales` set up. There is no save button in the Settings dialog; changes are automatically saved. I’ll improve the UX and update README 😅
